src/EnergyPlus/SystemReports.cc

```diff
@@ -2285,18 +2285,6 @@ namespace SystemReports {
                                     PrimaryAirSystem(SysIndex).Name);
             }
         }
-        SetupOutputVariable("Air System Relief Air Total Heat Loss Energy",
```

Comment:
And these were setup here but that moved, which is why we have a bunch of rdd diffs.